__tests__/app.test.js

```diff
@@ -70,6 +70,7 @@ describe("GET /api/articles/:article_id", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(String),
```

Comment:
we actually want a new standalone test for comment count!  The idea is that this previous test should be flexible to pass without expecting comment_count.